app/src/main/java/com/pr0gramm/app/ui/fragments/FeedFragment.java

```diff
@@ -10,25 +10,14 @@
 import android.support.v7.widget.RecyclerView;
 import android.support.v7.widget.SearchView;
 import android.util.Log;
-import android.view.Display;
-import android.view.LayoutInflater;
-import android.view.Menu;
-import android.view.MenuInflater;
-import android.view.MenuItem;
-import android.view.View;
```

Comment:
Hat geklappt, Anmerkungen sind im dritten commit drin. 
Der indicator wird jetzt dargestellt anschließend wird etwas verzögert der refresh ausgelöst (die Verzögerung ist nötigt weil sonst der indicator eher schlecht aussieht).
Zusätzlich ist in den Settings ein Eintrag wo der Button deaktiviert werden kann.